fc/jira/issue.py

```diff
@@ -117,20 +116,21 @@ def get_issue(cls, issue_id: str, auth: Auth) -> 'Issue':
         except HTTPError as exception:
             raise TaskException('Invalid issue key {}'.format(issue_id)) from exception
 
-        type = issue_json['fields']['issuetype']['name']
-        if type != 'Task' and type != 'Story' and type != 'Triage Task':
-            issue = cls()
-            issue.from_json(issue_json, auth)
-        elif type == 'Story':
+        issue_type = issue_json['fields']['issuetype']['name']
+
+        if issue_type == 'Story':
             issue = BacklogStory.from_json(issue_json, auth)
-        elif type == 'Task':
+        elif issue_type == 'Task':
             issue = BacklogTask.from_json(issue_json, auth)
-        else:
+        elif issue_type == 'Triage Task':
             labels: list = issue_json['fields']['labels']
             if 'EL' in labels:
                 issue = ElTask.from_json(issue_json, auth)
             else:
                 issue = TriageTask.from_json(issue_json, auth)
+        else:
+            issue = cls()
+            issue.from_json(issue_json, auth)
```

Comment:
So, as you said, other project stories (and tasks) will create a `BacklogStory` or a `BacklogTask`.  Do you think we should interrogate the `issue_json['fields']['project']['key']` here to determine whether to build just an `Issue` versus one of the aforementioned types?

I'm thinking we should do it here since since we do the interrogation once in one place (here) versus in many places.  Doing it here would also make it so we don't have incorrect `Backlog_` object.  I would hate to call the `score` method on a `BacklogStory` which is actually a `QPPCT` (or `QPPA`, or...) story that doesn't have any concept of scoring.

Reply:
i did add code to throw an error if a user tries to call `backlog score` on something not `QPPFC`, but that's a good question... perhaps we need to do checks across the board.  Things like changing status really aren't transferable across projects because each project and type can have its own workflow scheme.